### PR TITLE
Feature/mvp with jaccard

### PR DESCRIPTION
**Pull request by misharigot**
- Added a working MVP with Jaccard similarity with the required output format.
- Fixed tests.
- Added naive end-to-end test.

### CODE REVIEW

File: src/starter_code.py [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
 import pandas as pd
+from elasticsearch import Elasticsearch
 from tqdm import tqdm
 
+import entity_linking as el
 import information_extraction as ie
 from nlp_preprocessing import preprocess_text
 
-KEYNAME = "WARC-Record-ID"
+KEYNAME = "WARC-TREC-ID"
```
Comment (kevinh1998) [inline src/starter_code.py:15]: Needs to be 'Record-ID'

File: src/starter_code.py [hunk 2]
```diff
@@ -2,18 +2,23 @@
 import gzip
 import pstats
 import sys
+from typing import Tuple
 
 import pandas as pd
+from elasticsearch import Elasticsearch
```
Comment (kevinh1998) [inline src/starter_code.py:8]: Import doesn't seem to be used

**Review by kevinh1998 — COMMENTED** [review]
Two small remarks

**Comment by misharigot** [comment]
Can only use some code from here, but no longer able to merge, too much of a hassle.

**Pull request closed without merging**